src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -14,6 +28,24 @@
 
     @Override
     public int getSize() {
-        return 0;
+        return size;
+    }
+
+    private void resize() {
+        Node<K, V>[] currentTable = table;
+        table = (Node<K, V>[]) new Node[currentTable.length * RESIZE_NUMBER];
+
+    }
+
+    private static class Node<K, V> {
+        final K key;
+        final V value;
```

Comment:
Don't use getters and setters in class Node. Direct access to the fields is acceptable since it's a private inner class.